### DISCUSSION TITLE
第二设备手机连接了但无法使用

### DISCUSSION THREAD

**Discussion by KxiaoluoK** [getlantern/lantern-forum-cn#985] [Q&A]
1.用第一设备mac可以顺利上网，第二设备三星手机s23登同一账号不可以，显示已连接但google play/youtube都上不了
2.手机版的“账号管理”里，显示已有设备只有mac，没有手机。而电脑版没有找到已授权设备在哪里。不知是不是添加第二设备的步骤出了问题。
3.你们手机端的报告问题太难提交了，总是error。显示strconv.Atoi:parsing"":invalid syntax。以为是某些特殊字符的原因试着改了好几次还是不行，只能在这儿问了。
4. 如果还需要什么信息请联系我的邮箱kxiaoluok@126.com

**Comment by Raymondlee321** [comment]
我的问题跟你一样，电脑上的倒还勉强能用，就是时不时要等等，还是能打开的。可是手机端真的是无力吐槽，google play和telegram都用不了。

**Comment by MaxMKMax** [comment]
@KxiaoluoK 谢谢您提供的信息. 我会帮您反馈给技术团队. 不过, 如果可以的话您帮我提交以下应用内报告, 用电脑提交也可以.

**Reply by KxiaoluoK** [reply]
已提交，other {78391269}

**Reply by KxiaoluoK** [reply]
> @KxiaoluoK 谢谢您提供的信息. 我会帮您反馈给技术团队. 不过, 如果可以的话您帮我提交以下应用内报告, 用电脑提交也可以.

@Mike777Khoo 请问此问题有什么进展吗？

**Comment by MaxMKMax** [comment]
@Raymondlee321 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
